source/expressions.tex

```diff
@@ -5582,39 +5582,35 @@
 \end{bnf}
 
 \pnum
-The binary operator \tcode{.*} binds its second operand, which shall be
-of type ``pointer to member of \tcode{T}'' to its first operand, which shall be
-a glvalue
-of
-class \tcode{T} or of a class of which \tcode{T} is an unambiguous and
-accessible base class. The result is an object or a function of the type
-specified by the second operand.
-
-\pnum
-The binary operator \tcode{->*} binds its second operand, which shall be
-of type ``pointer to member of \tcode{T}'' to its first operand, which shall be of
-type ``pointer to \tcode{U}''
-where \tcode{U} is either \tcode{T} or
-a class of which \tcode{T}
-is an unambiguous and accessible base class.
-The expression \tcode{E1->*E2} is converted into the equivalent form
-\tcode{(*(E1)).*E2}.
+In a pointer-to-member expression of the form \tcode{E1->*E2}, \tcode{E2} shall
+be of type ``pointer to member of \tcode{T}'' and \tcode{E1} shall be of type
+``pointer to \cv{}~\tcode{U}'' where \tcode{U} is \tcode{T} or a type of which
+\tcode{T} is an accessible and unambiguous base class. The expression is converted
```

Comment:
This seems to be redundant with the checks after the rewrite.

Reply:
My intent was to allow for the reader to clearly reason the requirements on the operands, but this could be removed in favor of simply saying:
> In a pointer-to-member expression of the form `E1->*E2`,  `E1` shall be a prvalue of pointer to object type. The expression is converted to the equivalent form `(*(E1)).*E2`; the remainder of this subclause will
address only the operator `.*`.